### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by hectormmg**
This PR:
- Removes question issue template
- Adds disclaimers about bug issues without logs or network traces getting deleted without comment
- Adds a link to a network trace collection guide and a warning about properly scrubbing network traces.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
-    value: "## Security Reporting"
-- type: markdown
-  attributes:
-    value: |
-      If you find a security issue with our libraries or services [please report it to the Microsoft Security Response Center (MSRC)](https://aka.ms/report-security-issue) with as much detail as possible. Your submission may be eligible for a bounty through the [Microsoft Bounty](http://aka.ms/bugbounty) program. Please do not post security issues to GitHub Issues or any other public site. We will contact you shortly upon receiving the information. We encourage you to get notifications of when security incidents occur by visiting [this page](https://www.microsoft.com/msrc/technical-security-notifications) and subscribing to Security Advisory Alerts.
+    - type: markdown
+      attributes:
+          value: |
+              ## Before Submitting your Issue
+              Our GitHub issues are meant exclusively for bug reports. All issues regarding usage questions will be closed without comment. If you have a quesiton, please check to see if your question is answered in the FAQ for the library you are using:
```
Comment (hectormmg) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: This paragraph was updated.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 2]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+              - [@azure/msal-browser](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/FAQ.md)
+              - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md)
+              - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/FAQ.md)
+              - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+              - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/lib/msal-core/docs/FAQ.md)
+
+              In the case your question is not answered in the FAQ, please verify your question has not been covered on [Stack Overflow](https://stackoverflow.com/questions/tagged/msal.js).
```
Comment (hectormmg) [inline .github/ISSUE_TEMPLATE/bug_report.yml:16]: This paragraph was updated.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 3]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
+          label: Msal Logs
+          description: "Please provide verbose level log messages from msal. Bug reports without verbose MSAL logs will be closed without comment."
+
+    - type: textarea
+      attributes:
```
Comment (hectormmg) [inline .github/ISSUE_TEMPLATE/bug_report.yml:91]: This section was added

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 4]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+      validations:
+          required: true
+
+    - type: textarea
+      attributes:
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
```
Comment (hectormmg) [inline .github/ISSUE_TEMPLATE/bug_report.yml:86]: This section was updated.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+          label: Sanitized Network Trace
+          description: "Please upload a sanitized network trace (preferably from Fiddler, but network .HAR files are acceptable) containing the requests and responses from the failed authentication scenario. **Important: Please make sure to properly scrub all sensitive information from the trace, including but not limited to access/refresh tokens, client secrets, identifiable user information, etc.** Bug reports without sanitized network traces can be closed without comment. [Here's a guide](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-troubleshoot-network-trace) on how to collect a network trace."
+
+    - type: textarea
+      attributes:
+          label: MSAL Configuration
+          description: "Please provide your MSAL configuration. Bug reports without MSAL Configuration will be closed without comment."
```
Comment (hectormmg) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Description was updated

**Review by hectormmg — COMMENTED** [review]
Added comments on updated/added sections since formatter changes messed up the diff.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 1 above)
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Let's soften the wording here a bit
```suggestion
              Our GitHub issues are meant exclusively for bug reports and feature requests. Usage questions may be closed without comment. If you have a question, please check to see if your question is answered in the FAQ for the library you are using:
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 6]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+
+    - type: textarea
+      attributes:
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
+          label: Msal Logs
+          description: "Please provide verbose level log messages from msal. Bug reports without verbose MSAL logs will be closed without comment."
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
          description: "Please provide verbose or trace level log messages from msal. Bug reports without logs will be closed without comment."
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 7]
```diff
@@ -2,185 +2,193 @@ name: Bug Report
...
+
+    - type: textarea
+      attributes:
+          label: Sanitized Network Trace
+          description: "Please upload a sanitized network trace (preferably from Fiddler, but network .HAR files are acceptable) containing the requests and responses from the failed authentication scenario. **Important: Please make sure to properly scrub all sensitive information from the trace, including but not limited to access/refresh tokens, client secrets, identifiable user information, etc.** Bug reports without sanitized network traces can be closed without comment. [Here's a guide](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-troubleshoot-network-trace) on how to collect a network trace."
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Let's make sure we have instructions on how to scrub traces before we start asking for this.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (sameerag) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Addind docs here for logging instructions would help.

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (sameerag) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: +1

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 6 above)
Comment (hectormmg) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Done

File: .github/ISSUE_TEMPLATE/bug_report.yml (hunk 7 above)
Comment (hectormmg) [reply .github/ISSUE_TEMPLATE/bug_report.yml]: Removed for now.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 8]
```diff
@@ -2,185 +2,188 @@ name: Bug Report
...
+
+    - type: textarea
+      attributes:
+          label: Error Message
+          description: "Please provide any error message you are receiving and a stack trace."
+
+    - type: textarea
+      attributes:
+          label: Msal Logs
+          description: "Please provide verbose level log messages from msal. Bug reports without verbose MSAL logs will be closed without comment. Please look at our [Logging Guide](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/f9b96af818fd2725b8a8d93df7a9968492b59030/lib/msal-browser/docs/logging.md#L1) for instructions on how to collect MSAL logs."
```
Comment (tnorling) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
          description: "Please provide verbose or trace level log messages from msal. Bug reports without verbose or trace MSAL logs will be closed without comment. Please look at our [Logging Guide](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-browser/docs/logging.md) for instructions on how to collect MSAL logs."
```

**Review by tnorling — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 9]
```diff
@@ -2,185 +2,188 @@ name: Bug Report
...
+
+    - type: textarea
+      attributes:
+          label: Relevant Code Snippets
+          description: "Please provide relevant code snippets that can be used to reproduce this issue."
+          render: javascript
+      validations:
+          required: true
+
+    - type: textarea
```
Comment (sameerag) [inline .github/ISSUE_TEMPLATE/bug_report.yml:124]: Can we also add the without repro steps we also cannot address an issue?

**Review by sameerag — APPROVED** [review]
I assume the will be a new PR for mandating logs and fiddler traces once we have the guide ready. Approved with one comment.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 10]
```diff
@@ -2,185 +2,188 @@ name: Bug Report
...
+      attributes:
+          value: |
+              ## Issue Details
+
+    - type: dropdown
+      attributes:
+          label: Core Library
+          options:
+              - "MSAL.js (@azure/msal-browser)"
+              - "MSAL.js v1 (@azure/msal or msal)"
```
Comment (konstantin-msft) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Should we remove `MSAL.js v1`, `ADAL Node` and `ADAL.js` options?

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 11]
```diff
@@ -2,185 +2,188 @@ name: Bug Report
...
+    - type: dropdown
+      attributes:
+          label: Browsers Affected (Select all that apply)
+          multiple: true
+          options:
+              - Chrome
+              - Firefox
+              - Edge
+              - Safari
+              - Internet Explorer
```
Comment (konstantin-msft) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Should we remove `Internet Explorer` option as no longer supported?

**Review by konstantin-msft — APPROVED** [review]

**Review by peterzenz — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 12]
```diff
@@ -2,185 +2,189 @@ name: Bug Report
...
+              - [@azure/msal-node](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-node/docs/faq.md)
+              - [@azure/msal-angular](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-angular/docs/FAQ.md)
+              - [@azure/msal-react](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/lib/msal-react/FAQ.md)
+              - [@azure/msal or msal](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/msal-lts/lib/msal-core/docs/FAQ.md)
+
+              In the case your question is not answered in the FAQ, please verify your question has not been covered on [Stack Overflow](https://stackoverflow.com/questions/tagged/msal.js).
+              If you can't find a similar question on Stack Overflow, please create a new Stack Overflow thread using the tag `msal.js`.
+
+              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines may be closed without comment.
```
Comment (EmLauber) [inline .github/ISSUE_TEMPLATE/bug_report.yml]: Something about how they can still ask for timeline updates on existing feature request issues though
```suggestion
              If you have questions about timelines, please see our [roadmap](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/dev/roadmap.md). Issues pertaining to timelines without an explicit feature request may be closed without comment.
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 13]
```diff
@@ -2,185 +2,189 @@ name: Bug Report
...
+      attributes:
+          label: Wrapper Library Version
+          description: "Please enter the latest version this issue can be reproduced in. If you are not using a wrapper library please enter 'None'"
+          placeholder: "2.0.0"
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Public or Confidential Client?
```
Comment (EmLauber) [inline .github/ISSUE_TEMPLATE/bug_report.yml:64]: do we need a link off to explain public or confidential client? I feel like we get a lot of mis-matches here.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 14]
```diff
@@ -2,185 +2,189 @@ name: Bug Report
...
+      attributes:
+          label: Network Trace (Preferrably Fiddler)
+          description: "Please send a [Fiddler trace](https://learn.microsoft.com/en-us/azure/azure-web-pubsub/howto-troubleshoot-network-trace) (strongly preferred) or .har file containing the relevant network requests and responses to msaljshelp@microsoft.com. Your are reponsible for properly sanitizing any network trace you send. **DO NOT post a network trace**, sanitized or not to this or any GitHub issue. GitHub issues for which we don't receive a corresponding network trace e-mail in a reasonable timeframe may be closed without comment."
+          options:
+              - label: Sent
+              - label: Pending
+    - type: textarea
+      attributes:
+          label: MSAL Configuration
+          description: "Please provide your MSAL configuration. Bug reports without MSAL Configuration will be closed without comment."
```
Comment (EmLauber) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
          description: "Please provide your MSAL configuration. Bug reports without MSAL configuration will be closed without comment."
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 15]
```diff
@@ -2,185 +2,189 @@ name: Bug Report
...
+          label: Expected Behavior
+          description: "Please describe what you expect the behavior to be."
+      validations:
+          required: true
+
+    - type: dropdown
+      attributes:
+          label: Identity Provider
+          options:
+              - Azure AD / MSA
```
Comment (EmLauber) [inline .github/ISSUE_TEMPLATE/bug_report.yml]:
```suggestion
              - Entra ID (formerly Azure AD) / MSA
```

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 16]
```diff
@@ -2,185 +2,189 @@ name: Bug Report
...
+              - Azure B2C Basic Policy
+              - Azure B2C Custom Policy
+              - ADFS
```
Comment (EmLauber) [inline .github/ISSUE_TEMPLATE/bug_report.yml:149]: should we spell out what ADFS stands for?

Comment (hectormmg) [reply .github/ISSUE_TEMPLATE/bug_report.yml:149]: I feel like if they use ADFS they know what it means, right?

**Pull request merged**